recipes-devtools/visionworks/libvisionworks_1.6.0.501.bb

```diff
@@ -27,15 +27,15 @@ do_install() {
     install -d ${D}${prefix} ${D}${libdir} ${D}${datadir} ${D}${libdir}/pkgconfig
     cp -R --preserve=mode,timestamps ${B}/usr/include ${D}${prefix}/
     cp -R --preserve=mode,timestamps ${B}/usr/lib/pkgconfig/* ${D}${libdir}/pkgconfig
-    cp --preserve=mode,timestamps ${B}/usr/lib/libvisionworks.so ${D}${libdir}/
+    cp --preserve=mode,timestamps ${B}/usr/lib/libvisionworks.so* ${D}${libdir}/
     cp -R --preserve=mode,timestamps ${B}/usr/share/visionworks ${D}${datadir}/
 }
 
 INHIBIT_PACKAGE_STRIP = "1"
 INHIBIT_PACKAGE_DEBUG_SPLIT = "1"
 INHIBIT_SYSROOT_STRIP = "1"
 
-FILES_${PN} = "${libdir}/libvisionworks.so"
+FILES_${PN} = "${libdir}/libvisionworks.so*"
 FILES_${PN}-dev = "${includedir} ${libdir}/pkgconfig ${datadir}/visionworks"
```

Comment:
This line should be changed to
```
FILES_${PN}-dev += "${datadir}/visionworks"
```
the other directories mentioned should already get included by normal packaging, as will the ${SOLIBSDEV} symlinks in ${libdir} for the libraries.